README.md

```diff
@@ -35,7 +35,7 @@ Lightdash is a BI tool that is fully integrated with your dbt project, allowing
 * [x] 👩‍💻 Declare dimensions and metrics in yaml alongside your dbt project
 * [x] 🤖 Automatically creates dimensions from your dbt models
 * [x] 📖 All dbt descriptions synced for your users
-* [x] 📊 Simple data visualisations for your metrics
+* [x] 📊 Simple data visualizations for your metrics
```

Comment:
Still need to write a contributors guide but we use US English in the code base. so *-ize* is preferred.